cypress/e2e/lifetime.cy.ts

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+export {};
+
+import bytes from 'bytes';
+
+const GENERAL_COUPON_DISCOUNT = 'IoYrRdmY';
+const DRIVE_WEB_URL = Cypress.env('DRIVE_WEB_URL');
+const API_DRIVE_URL = Cypress.env('API_DRIVE_URL');
+
+const url = ({ planId, couponCode }: { planId: string; couponCode?: string }) => {
+  return `${DRIVE_WEB_URL}/new?redirectUrl=${encodeURIComponent(DRIVE_WEB_URL + '/checkout-plan')}${encodeURIComponent(
+    `?planId=${planId}${couponCode ? '&couponCode=' + couponCode : ''}&mode=payment`,
+  )}`;
+};
+
+//Check if the buttons works properly
+describe('Lifetime page', () => {
+  describe('When the payment lifetime button is clicked', () => {
+    describe('When the plan of 2TB is clicked', () => {
+      it('Redirect to stripe checkout with the correct planId and mode', () => {
+        cy.visit('/lifetime');
+        cy.get('#priceTable').contains('Get 2TB').click();
```

Comment:
Could we store this as a constant or enum or something that directly depends on the products? Then, we can avoid having to change each test that changes the label. So if you have in some .json or in some file this text, and is used for the component, let's use it here too, then, when something changes, you just need to change it in one place and not in 7.